### DISCUSSION TITLE
Output on Serial monitor

### DISCUSSION THREAD

**Discussion by jnivard** [Bodmer/TFT_eSPI#2475] [General]
Hai

When I use the TFT_eSPI library I get output on the serial.monitor of inpritable charackters and I can not get them readable by changing the boud rate.

Is there a explanation for that 

John

**Comment by Bodmer** [comment]
Maybe this is down to the pin allocations. I have not seen this issue.

I need more detailed information on your hardware, setup files and an example sketch that demonstrates the problem.

**Reply by jnivard** [reply]
Hai 

I am using the following pin allocation. Afther booting the serial monitor start outputing unprintable charackters. The booting itself is readable. 

#define TOUCH_CS 15 //
#define TFT_MISO 22 // 
#define TFT_MOSI 1 //
#define TFT_SCLK 23//
#define TFT_CS 2 //
#define TFT_DC 16 //
#define TFT_RST 4  //

Thanks

**Comment by Bodmer** [comment]
I am guessing you are using an ESP32.

Avoid using pins marked red in table here for the TFT:
https://randomnerdtutorials.com/esp32-pinout-reference-gpios/
